Assets/Character/Scripts/Consumable1.cs

```diff
@@ -0,0 +1,49 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Consumable1 : MonoBehaviour {
+
+    private int count;
+    private float cooldown;
+    private float startTime;
+    private bool healing;
+    private float duration;
```

Comment:
it makes more sense to define a total health and the time it takes to finish.

For example, a health potion can heal 100 hp over 10 seconds. 
Or a health potion can heal 30 hp over 2 seconds. etc etc